remix/app/helpers/localStorage.helper.ts

```diff
@@ -0,0 +1,18 @@
+import { Notification } from '../types';
+
+export function addNotification({ text, type }: Notification) {
+  localStorage.setItem('notificationText', text);
+  localStorage.setItem('notificationType', type as string);
```

Comment:
Based on the comment to refactor using an object (https://github.com/nimblehq/js-framework-benchmark/pull/120#discussion_r1226039076), I expected you to revisit the LocalStorage implementation, too 😅 

```suggestion
  localStorage.setItem('notification', { text, type });
```

Reply:
For this one, I know that storing the object on the `localStorage` use `JSON.stringify()`, and `JSON parse` for retrieving the data. That's why I think storing `key`+`value` one by one might be simpler.

But in terms of structuring the `localStorage` by wrapping the object into one will be a better approach.

Improved here: [5b3237a](https://github.com/nimblehq/js-framework-benchmark/pull/120/commits/5b3237af21e9837710fef39f565836071a15e44b)